src/main/telemetry/smartport.c

```diff
@@ -379,8 +377,10 @@ void handleSmartPortTelemetry(void)
                 }
                 break;
             case FSSP_DATAID_HEADING    :
-                smartPortSendPackage(id, attitude.values.yaw * 10); // given in 10*deg, requested in 10000 = 100 deg
-                smartPortHasRequest = 0;
+		if (sensors(SENSOR_MAG)) {
```

Comment:
Why? heading is available even without magnetometer (with usable short-term relative accuracy)